packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -263,13 +262,172 @@ function joinMeeting(meeting) {
   });
 }
 
-// In order to simplify the state management needed to keep track of our button
-// handlers, we'll rely on the current meeting global object and only hook up event
-// handlers once.
+function handleRadio() {
+  // checks which radio is enabled
+  isPaste = document.getElementById('paste').checked;
+  isOAuth = document.getElementById('oauth').checked;
+
+  const div = document.createElement('div');
+
+  // removes the child node of the form if anything exists
+  while (document.getElementById('authForm').firstChild) {
+    document.getElementById('authForm').removeChild(document.getElementById('authForm').firstChild);
+  }
+
+  // adds the correct form based on enabled radio
+  if (isOAuth) {
+    const legend = document.createElement('legend');
```

Comment:
Please move this into the `index.html` file and use
`document.getElementById("myDIV").style.display = "none";` to hide/show it.

Reply:
Weird this is an old commit. This isn't right